packages/admin/src/Resources/Resource.php

```diff
@@ -368,11 +368,11 @@ public static function getSlug(): string
             ->implode('/');
     }
 
-    public static function getUrl($name = 'index', $params = []): string
+    public static function getUrl($name = 'index', $params = [], $absolute = true): string
```

Comment:
```suggestion
    public static function getUrl($name = 'index', $params = [], $isAbsolute = true): string
```

Reply:
Yeah but since this code is in the Filament domain, we should keep the parameter names consistent with the rest of our code.

Reply:
Agree, changed it!